stage #4/module #5. AWS/theory/dynamodb.md

```diff
@@ -0,0 +1,146 @@
+# AWS DynamoDB
+
+### Recommended timeline
+The recommended timeline for the whole dynamoDB part is approximately 6 hours.
```

Comment:
is it possible to create a general file that will describe the order of topics to study, as well as the recommended timeline for the whole module?

Reply:
Sure!